apps/web/modules/ee/sso/lib/sso-handlers.ts

```diff
@@ -31,12 +32,33 @@ export const handleSsoCallback = async ({
   account: Account;
   callbackUrl: string;
 }) => {
+  const debugContext = createDebugContext(user, account, callbackUrl);
```

Comment:
I like that we see the context once, but it seeing the full context every time creates a lot of noise in the logs. I would suggest that we log the full context only once in the beginning of handleSsoCallback function. Having a context to know what the logs belong to is still important but maybe one identifier is enough and we don't need the full user + account?